config/terms.yml

```
@@ -59,6 +59,12 @@
     embargo_history: embargoHistory
     visibility_after_embargo: visibilityAfterEmbargo
 
+# Trying to make ::RDF::Vocab::RDAU.P60550 less opaque
+- vocabulary: rdau
+  schema: 'http://rdaregistry.info/Elements/u/'
+  terms:
+    extent: extent.en
```

I just looked back at the FolkFestTriples - v.1' and the Data Dictionary.  The URI as specified was 'http://rdaregistry.info/Elements/u/P60550'

```suggestion
    extent: P60550
```